### DISCUSSION TITLE
Usage with nodejs

### DISCUSSION THREAD

**Discussion by bzuker** [safe-global/safe-apps-sdk#253] [Q&A]
Would it be possible to use this package in node?
I saw that there are a couple of dependencies on `window`, so I’m guessing the initial answer would be no. 

Is there something like a wrapper for the API calls possible to the gnosis relayer?

thanks in advance!

**Comment by mmv08** [comment]
> Would it be possible to use this package in node? I saw that there are a couple of dependencies on `window`, so I’m guessing the initial answer would be no.
> 
> Is there something like a wrapper for the API calls possible to the gnosis relayer?
> 
> thanks in advance!

This library is intended to be used for Safe Apps, which live inside an iframe, so no, the package is not designed for use with node.js.

We also don't have a relayer at the moment, except one for the legacy wallet
